src/router.ts

```diff
@@ -90,6 +91,19 @@ export class Router {
       const thresholdPrice = blackScholes(currentPrice, dip_deposit.strike / 1_000_000, fractionOfYear, vol, 0.01, 'call');
 
       const { price } = order;
+      const userPremium = price * dip_deposit.qty;
+      if (userPremium < minExecutionPremium) {
+        // If user premium is too small don't bother spamming MM
+        console.log('Not routing too small of a trade:', userPremium, minExecutionPremium);
+        this.dips[
+          this.dip_to_string(
+            dip_deposit.expirationMs / 1_000,
+            dip_deposit.strike,
+          )
+        ] = dip_deposit;
+        this.run_risk_manager();
+        return;
+      }
       // TODO: Test this to make sure the decimals are correct on each.
       console.log('MM price:', price, 'BVE price:', thresholdPrice);
```

Comment:
move the log line above your block of code

Reply:
96 or 107?

Reply:
console.log('MM price:', price, 'BVE price:', thresholdPrice);

Reply:
want the mm price to be logged before the first time we make a decision based on it